editor/i18n/en/localization.js

```diff
@@ -377,6 +389,17 @@ module.exports = {
         handleTouchEvent:
             'Once checked, the RichText will block all input events (mouse and touch) within the bounding box of the node, preventing the input from penetrating into the underlying node.',
     },
+    UICoordinateTracker: {
+        target: 'Target node',
+        camera: '3D camera for conversion',
+        use_scale: 'Do you need to scale the converted 2d node size according to how far the 3D node is from the camera',
```

Comment:
Whether to scale the converted 2d node's size according to the distance between the camera and the 3d node.